docs/source/run/service.rst

```diff
@@ -0,0 +1,93 @@
+====================
+Running as a service
+====================
+
+Now that you can run the bot, you probably want it to run forever in the
+background of your system. While there are many ways to do so, there are a few
+steps and requirements that remain the same to properly run Sopel as a service.
+
+
+Requirements to run a service
+=============================
+
+Sopel needs the following:
+
+* a non-root user to run as
+* read/write access to its configuration file
+* a writable pid directory
+  (:attr:`~sopel.config.core_section.CoreSection.pid_dir`)
+* a writable logs directory
+  (:attr:`~sopel.config.core_section.CoreSection.logdir`)
+* a writable data directory
+  (:attr:`~sopel.config.core_section.CoreSection.homedir`)
+
+.. important::
+
+    Do **not** run Sopel as the **root** user. And avoid if possible to run
+    with a privileged user.
+
+
+Running with systemd
+====================
+
+`systemd`__ is a service manager used by Debian and most Debian-derived
+Linux distributions. Here is an example of a service unit file to run Sopel as
```

Comment:
Nit: might as well refer to Ubuntu here since it's a soft default elsewhere in this changeset

```suggestion
`systemd`__ is a service manager used by Ubuntu and most Debian-derived
Linux distributions. Here is an example of a service unit file to run Sopel as
```